sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items.ipynb

```diff
@@ -375,4 +495,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 4
```

Comment:
Please add a text/code section on how to see the HyperDrive parent run. In v1 notebook, we have this after the "automl_image_run.wait_for_completion(wait_post_processing=True)" cell.
For the code part, you can refer to "Hyperparameter sweeping for image tasks" of this tutorial https://docs.microsoft.com/en-us/azure/machine-learning/tutorial-auto-train-image-models?tabs=SDK-v2%2CSDK-v2-#hyperparameter-sweeping-for-image-tasks